client/package.json

```diff
@@ -14,6 +14,7 @@
     "elkjs-7-1": "npm:elkjs@0.7.1",
     "elkjs-8": "npm:elkjs@0.8.2",
     "elkjs-9": "npm:elkjs@0.9.0",
+    "elkjs-9-1": "npm:elkjs@0.9.1",
```

Comment:
I think it would be sufficient to only lit elkjs 0.9.0 here, since it is the same as 0.9.0. Also see below.